### PR TITLE
Log query params when available

### PR DESCRIPTION
**Pull request by JulienPradet**
We are doing many requests on the same URL but with different params.
We'd find it useful to have query params to easily reproduce the URLs we want to debug.

This is what this PR stands for. :slightly_smiling_face: 

Have a nice day!

### CODE REVIEW

**Comment by Gerhut** [comment]
Hi @JulienPradet 

Thanks for your contribution!

Did you try customizing the logger refer to <https://github.com/Gerhut/axios-debug-log#configuration> or proposed that this should be the default behavior?

**Comment by JulienPradet** [comment]
@Gerhut  I suggest that this should be the default.
In my experience, the URL without the GET params is not really useful. But I don't know if it is relevant for you.

File: index.js [hunk 1]
```diff
@@ -2,22 +2,35 @@
 
 var axios = require('axios')
 var axiosDebug = require('debug')('axios')
+var querystring = require('querystring')
```
Comment (Gerhut) [inline index.js]: I have a concern that if the querystring module is good here due to axios-debug-log is designed to be an isomorphic module. After querystring module is referenced, user could just use it in node.js or module bundler including polyfills of node.js like webpack.

File: test.js [hunk 2]
```diff
@@ -78,6 +95,48 @@ it('should logging request of axios instance', () => axios.create()({
...
+}).then(() => {
+  debug.log.should.be.calledTwice()
+  debug.log.firstCall.should.be.calledWithExactly(
+    'GET http://example.com/?parameter%20name=%26'
+  )
+  debug.log.secondCall.should.be.calledWithExactly(
+    '200 BAR', '(GET http://example.com/?parameter%20name=%26)'
+  )
+}))
+
```
Comment (Gerhut) [inline test.js:186]: I am not sure what happends if `url: 'http://example.com/?foo=bar'` and `params: { baz: 'qux' }`

File: index.js (hunk 1 above)
Comment (Gerhut) [reply index.js]: Actually I wonder if we can borrow this part from axios itself to ensure the log and the actual request are the same, even axios have some bug.

Comment (JulienPradet) [reply index.js]: Ah right! I've pushed a new commit using axios's `buildURL` helper. In the [latest version which is still in beta](https://github.com/axios/axios/releases/tag/v0.19.0-beta.1), axios will expose a [`getUri`](https://github.com/axios/axios/pull/1712) method, but I didn't use it to make sure that the debugger still works with older versions.

This should also resolve your other comment (which I didn't think about, thanks :) ).

**Pull request merged**